common/persistence/dataVisibilityManagerInterfaces.go

```diff
@@ -138,6 +138,8 @@ type (
 		ListWorkflowExecutionsRequest
 		PartialMatch        bool
 		WorkflowSearchValue string // This value will be searched across workflow type, workflow ID and runID
+		SortColumn          string // This should be a valid search attribute
+		SortOrder           string // DESC or ASC
```

Comment:
can this be typed instead? Otherwise it requires this comment pretty everywhere it gets mentioned...

Reply:
fair point. but unsure where would be a common point to create this type since we have the request definition in the persistence layer and in the api tyoes layer